framework-v1/README.md

```diff
@@ -17,7 +17,7 @@ Links:
   - [voting details](./elections-voting.md)
 - [NDC Product Book](https://docs.google.com/document/d/1w_wfRfp-ISH7g-zu7vAFULVvRNwyLGwNIDC1EBkxvu0)
 - [Gov Ops Manual](https://docs.google.com/document/d/1l5g7JhaEPUMzjrzXEKw2Za5UmuRyK_9qxj-2-2CxlOE/edit?usp=drivesdk)
```

Comment:
This ops manual is old link, is fixed below. Here is correct link:
https://near-ndc.notion.site/NDC-V1-Ops-Manual-914bf75675284daf8314e1a05c0a2eea